contracts/cw-named-groups/src/msg.rs

```diff
@@ -0,0 +1,58 @@
+use std::collections::HashSet;
+
+use cosmwasm_std::Addr;
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+pub struct Group {
+    pub name: String,
+    pub addresses: HashSet<String>,
```

Comment:
When this comes in as JSON how does this look? Will the frontend just provide a list? Will deserialization fail if the provided list has duplicate values?

Reply:
Might be worthwhile to just take a `Vec<String>` in the instantiate message if these don't have clean answers. Can't find anything about this in the `serde_json` docs.

Reply:
Good catch! Should have been a `Vec` :)